trials/death/death.cpp

```diff
@@ -0,0 +1,13 @@
+//
+// Created by robert on 11/21/20.
+//
+#include <iostream>
+
+int main() {
+    for(int i = 0; i < 10; i++) {
```

Comment:
You can initialize a stupid int with auto if you like

Reply:
do you think that it is better to use "auto"?

Reply:
No. Not in this case. But if you were compariing against a size, which is often type size_t then you get an annoying conversion warning